Misc/STAR_BAKUR RFC Area.txt

```diff
@@ -1,5 +1,4 @@
 ;BAKUR RFC Area
-BAKUR RFC Area               N052.48.54.000 W005.30.00.000 N052.20.00.000 W005.30.00.000
-BAKUR RFC Area               N052.20.00.000 W005.30.00.000 N052.12.54.045 W005.43.48.964
-BAKUR RFC Area               N052.12.54.045 W005.43.48.964 N052.42.39.000 W005.58.31.000
-BAKUR RFC Area               N052.42.39.000 W005.58.31.000 N052.48.54.000 W005.30.00.000
+BAKUR RFC Area               N052.12.10.618 W005.45.13.479 N052.47.04.756 W006.03.01.739
+BAKUR RFC Area               N052.47.04.756 W006.03.01.739 N052.47.04.756 W006.03.01.739
```

Comment:
This line segment seems to start and end in the same place, I assume it can be removed, leaving just the two lines which do not follow the FIR boundary?